Gemfile.lock

```diff
@@ -362,19 +443,34 @@ GEM
       net-ssh (>= 2.7)
       net-telnet
       sfl
-    spoon (0.0.6)
-      ffi
+    stackdriver-core (1.3.0)
+      google-cloud-core (~> 1.2)
     syslog-logger (1.6.8)
     systemu (2.6.5)
     thor (0.20.0)
     timeliness (0.3.8)
     toml-rb (1.1.1)
       citrus (~> 3.0, > 3.0)
     tomlrb (1.2.7)
+    train (1.4.24)
+      aws-sdk (~> 2)
+      azure_mgmt_resources (~> 0.15)
+      docker-api (~> 1.26)
+      google-api-client (~> 0.19.8)
```

Comment:
I am cautiously optimistic that this will still allow `kitchen-google`—which also [requires `google-api-client`](https://github.com/test-kitchen/kitchen-google/blob/v1.5.0/kitchen-google.gemspec#L18)—to get `chef gem install`'d into a Workstation.